aea/cli/search.py

```diff
@@ -139,6 +140,38 @@ def connections(ctx: Context, query):
     print(format_items(sorted(result, key=lambda k: k["name"])))
 
 
+@search.command()
+@click.option("--query", default="", help="Query string to search Contracts by name.")
+@pass_ctx
+def contracts(ctx: Context, query):
+    """Search for Contracts."""
+    if ctx.config.get("is_registry"):
```

Comment:
Please refactor this and the other search commands do avoid code duplication and allow for code reuse.

Reply:
Rows 148-172 are generic apart from the `contracts` string afaikt

Reply:
Could have something like `_search_item(ctx, query, pluralized_item_type)`